### DISCUSSION TITLE
How to delete my account?

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [planetscale/discussion#196] [Q&A] [answered]
I could not find any option to delete my account.

**Comment by fooforge** [comment]
Hi @rohandebsarkar, self-serve account/org deletions is something that we're working on, but for the time being it requires our assistance. Create a ticket at https://support.planetscale.com/ from your registered email address and we'll help you out.

**Comment by jgreet** [comment, marked as answer]
User accounts can now deleted from the settings page: https://app.planetscale.com/settings/account

**Reply by anirudhuuu** [reply]
The option to delete from settings page is not available, has it been moved to a new location?

**Reply by mscoutermarsh** [reply]
Hi @jwala-anirudh, I just checked and I'm still seeing it. Could you send us a message here https://support.planetscale.com/ with your account info? We can check.
